victor/src/layout/flow/mod.rs

```diff
@@ -192,7 +192,6 @@ fn layout_block_level_children<'a>(
     }
     let content_block_size =
         current_block_direction_position + ongoing_collapsed_margin.context.end.solve();
-    let block_size = containing_block.block_size.auto_is(|| content_block_size);
```

Comment:
In https://github.com/SimonSapin/victor/commit/15bbe3cd861e246dc6a7542559fb42b03221120d I made this same change but had to move this operation elsewhere. Do reftests still pass?

Reply:
I do it in `layout_in_flow_non_replaced_block_level` but I forgot to even run the tests, will check.

Reply:
I fixed this.